video_creation/final_video.py

```diff
@@ -349,15 +351,13 @@ def on_update_example(progress):
         old_percentage = pbar.n
         pbar.update(status - old_percentage)
 
-    path = f"results/{subreddit}/{filename}"
-    path = path[:251]
```

Comment:
nope. 


revert this. It will cause 3 separate errors that are a pain to figure out.

Reply:
Ok, i'll change it. I don't know why should give a error but ok.

Reply:
It's like limiting the path length? So i should add path += ".mp4" too. Ok i got it

Reply:
Holy, this error should be very painful to find xD